docs/Manual/GettingStarted/kube-arangodb/README.md

```diff
@@ -0,0 +1,209 @@
+# Start ArangoDB on Kubernetes in 5min
+
+Starting an ArangoDB database (either single server or full blown cluster)
+on Kubernetes involves a lot of resources.
+
+The servers needs to run in Pods, you need secrets for authentication,
+TLS certificates and Services to enable communication with the database.
+
+Use `kube-arangodb`, the ArangoDB Kubernetes operator to greatly simplify
+this process.
+
+In this guide, we'll explain what the ArangoDB Kubernetes operator is,
+how to install it and how use it to deploy your first ArangoDB database
+in a Kubernetes cluster.
+
+## What is `kube-arangodb`
+
+`kube-arangodb` is a set of two operators that you deploy in your Kubernetes
+cluster to (1) manage deployments of the ArangoDB database and (2)
+provide `PersistenVolume's` on local storage of your nodes for optimal
+storage performace.
+
+Note that the operator that provides `PersistentVolume's` is not needed to
+run ArangoDB deployments. You can also use `PersistentVolume's` provided
+by other controllers.
+
+In this guide we'll focus on the `ArangoDeployment` operator.
+
+## Installing `kube-arangodb`
+
+To install `kube-arangodb` in your Kubernetes cluster, make sure
+you have acces to this cluster and the rights to deployment resources
+at cluster level.
+
+For now, any recent Kubernetes cluster will do (e.g. `minikube`).
+
+Then run (replace `<version>` with the version of the operator that you want to install):
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/arangodb/kube-arangodb/<version>/manifests/crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/arangodb/kube-arangodb/<version>/manifests/arango-deployment.yaml
+```
+
+The first command installs two `CustomResourceDefinition's` in your Kubernetes cluster:
+
+- `ArangoDeployment` is the resource used to deploy ArangoDB database.
+- `ArangoLocalStorage` is the resource used to provision `PersistentVolume's` on local storage.
+
+The second command installs a `Deployment` that runs the operator that controls
+`ArangoDeployment` resources.
+
+## Deploying your first ArangoDB database
+
+The first database we're going to deploy is a single server database.
+
+Create a file called `single-server.yaml` with the following content.
+
+```yaml
+apiVersion: "database.arangodb.com/v1alpha"
+kind: "ArangoDeployment"
+metadata:
+  name: "single-server"
+spec:
+  mode: single
+```
+
+Now insert this resource in your Kubernetes cluster using:
+
+```bash
+kubectl apply -f single-server.yaml
+```
+
+The `ArangoDeployment` operator in `kube-arangodb` will now inspect the
+resource you just deployed and start the process to run a single server database.
+
+To inspect the currentl status of your deployment, run:
+
+```bash
+kubectl describe ArangoDeployment single-server
+# or shorter
+kubectl describe arango single-server
+```
+
+To inspect the pods created for this deployment, run:
```

Comment:
no komma

Reply:
Don't like it without comma. Are you really sure?